server/routes/translationRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+
+import { translatingFunction } from '../utils/translatingFunction.js'
+
+const translationRouter = express.Router()
+
+translationRouter.route('/').get(translatingFunction)
```

Comment:
is this a get req? on the function you are extracting stuff from the body, and if there is a body - this should be post/put